Content.Server/StationEvents/Components/GameDirectorSystemComponent.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System.Runtime.Serialization;
+using Content.Shared.chaos;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(GameDirectorSystem))]
+public sealed class GameDirectorSystemComponent : Component
```

Comment:
rename to GameDirector/RoundDirector

Reply:
I prefer Game Director because it has a specific meaning. I'll lose the system.